src/components/Header.js

```diff
@@ -1,9 +1,23 @@
 import "./Header.css";
+import { Switch, Route } from "react-router-dom";
 
-export default function Header() {
+export default function Header({ text }) {
   return (
     <header className="Header">
-      <h1>My cakes app</h1>
+      <Switch>
+        <Route path="/cakes/:id">
+          <h1>Single MyCakes</h1>
+        </Route>
+        <Route path="/favs">
+          <h1>Favourites</h1>
+        </Route>
+        <Route path="/add">
+          <h1>Insert a new cake</h1>
+        </Route>
+        <Route exact path="/">
+          <h1>Check the cake!</h1>
```

Comment:
Maybe here you want to say, `Check the Cakes`?

Reply:
Still looking for a proper name for this App and pages :D